src/azure-cli/azure/cli/command_modules/backup/tests/latest/preparers.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, name_prefix='clitest-vault', parameter_name='vault_name',
                  resource_group_location_parameter_name='resource_group_location',
                  resource_group_parameter_name='resource_group',
                  dev_setting_name='AZURE_CLI_TEST_DEV_BACKUP_ACCT_NAME', soft_delete=True):
-        super(VaultPreparer, self).__init__(name_prefix, 24)
+        super().__init__(name_prefix, 24)
```

Comment:
This change doesn't belong to this PR.

Reply:
Ok. This is also one of the silenced pylint rules. Perhaps semgrep could automate fixing this rule.

Reply:
I moved it to #30367